carbon-design-system/carbon
# fix(FilterableMultiSelect): open item list when searching

Closes #5002 

This PR opens the item list for filterable multiselects when the input value changes so that the component will not be stuck in a state where users are modifying the input value and the item list will not appear

#### Testing / Reviewing

1. open the item list for the filterable multiselect
2. click on the input field to collapse the item list
3. start typing to trigger a search/filter action
4. observe that the list is not permanently closed
